pkg/models/model_plan.go

```diff
@@ -19,8 +23,36 @@ func NewModelPlan(createdBy uuid.UUID, modelName string) *ModelPlan {
 		Archived:   false,
 		Status:     ModelStatusPlanDraft,
 	}
+}
 
+// NewModelPlanFull returns a new model plan with all fields set
+func NewModelPlanFull(
```

Comment:
I think we can get rid of this method now that it's not used in the storage methods anymore!